server/manual-rendering.md

```diff
@@ -0,0 +1,63 @@
+### Manual rendering
+
+To see how to use the template engine, we'll first do this outside of Express by using only JavaScript, Node.js and the `handlebars` npm package.
+
+{% raw %}
+```js
+var handlebars = require('handlebars')
+
+var simpleTemplate = '<p>{{para}}</p>'
+var createSimpleResult = handlebars.compile(simpleTemplate)
+var simpleResult = createSimpleResult(data)
+
+console.log(simpleResult)
+```
+{% endraw %}
+
+Notice how the `compile` step returns a function that accepts the data to apply to the template. How about the conditionals and iterations that was mentioned earlier?
+
+{% raw %}
+```js
+var handlebars = require('handlebars')
+
+var data = {
+  trueBool: true,
+  cool: 'okay, this is cool',
+  colours: ['red', 'green', 'blue'],
+  obj: {
+    prop: 'nested object property'
+  }
+}
+
+// conditional
+var conditionalTemplate = '<p>' +
+  '{{#if trueBool}}' +
+  'truthy :)' +
```

Comment:
I think the smiley faces make this code harder to read as students are still getting their heads around < { ( [ : ; . , ] ) } > symbols already